test/setup/README.md

```diff
@@ -0,0 +1,35 @@
+# Integration Testing
```

Comment:
Instead of maintaining this separately, could we also copy over the standard [Contributing file](https://github.com/terraform-google-modules/terraform-google-module-template/blob/master/terraform-google-%7B%7Bcookiecutter.module_name%7D%7D/CONTRIBUTING.md)?

Reply:
Added